go/engine/signup.go

```diff
@@ -239,11 +247,22 @@ func (s *SignupEngine) registerDevice(a libkb.LoginContext, ctx *Context, device
 	return nil
 }
 
-func (s *SignupEngine) genPaperKeys(ctx *Context) error {
+func (s *SignupEngine) genPaperKeys(ctx *Context, lctx libkb.LoginContext) error {
+	// Load me again so that keys will be up to date.
+	var err error
+	s.me, err = libkb.LoadUser(libkb.LoadUserArg{Self: true, UID: s.me.GetUID(), PublicKeyOptional: true, Contextified: libkb.NewContextified(s.G())})
```

Comment:
Would be nice to avoid the full user load here when a `UserPlusAllKeys` would suffice.